Spigot-Server-Patches/0223-Add-missing-coverages-for-getTileEntity-in-order-to-.patch

```diff
@@ -0,0 +1,43 @@
+From 8b3b1815fca493447bd38708ec9786d4e9f55117 Mon Sep 17 00:00:00 2001
+From: Shane Freeder <theboyetronic@gmail.com>
+Date: Sat, 22 Jul 2017 15:22:59 +0100
+Subject: [PATCH] Add missing coverages for getTileEntity in order to attempt
+ to avoid exeptions when calling getTileEntity
+
+
+diff --git a/src/main/java/net/minecraft/server/WorldServer.java b/src/main/java/net/minecraft/server/WorldServer.java
+index bde48b692..01166f496 100644
+--- a/src/main/java/net/minecraft/server/WorldServer.java
++++ b/src/main/java/net/minecraft/server/WorldServer.java
+@@ -234,6 +234,28 @@ public class WorldServer extends World implements IAsyncTaskHandler {
+                 result = fixTileEntity(pos, type, result);
+             }
+         }
++        // Paper Start - add TE fix checks for shulkers, see nms.BlockShulkerBox
++        else if (type == Blocks.WHITE_SHULKER_BOX ||
```

Comment:
Looks like all of this can be replaced with type instanceof BlockShulkerBox and solve the obfuscation issue.